package co.dalmope.usermicroservice.infraestructure.drivenadapters.jpa.exception;

public class PersonAlreadyExistsException extends RuntimeException {
    public PersonAlreadyExistsException() {
        super();
    }
}
